package.json

```diff
@@ -112,6 +112,7 @@
     "react-native": "0.63.4",
     "react-native-android-badge": "^0.4.3",
     "react-native-appearance": "^0.3.2",
+    "react-native-auto-scale-text": "^1.0.7",
```

Comment:
Not sure if we really need a npm dependency for that. `Text` component has `adjustsFontSizeToFit` prop which allows for nice scaling down of text.

Reply:
@mdjastrzebski Thanks for letting me know. Done with the changes.